src/aatoolbox/datasources/iri/iri_seasonal_forecast.py

```diff
@@ -158,33 +135,43 @@ def load_raw(self):
             )
             return ds
         except ValueError as err:
+            # TODO: Maybe print the traceback if this error can also
+            #  happen due to a missing NetCDF backend
```

Comment:
That is what is being done by having `from err` after the raise. Or do you mean smth else?

Reply:
Oops yeah, I've removed this comment now